core/rpc/src/chain/mod.rs

```diff
@@ -81,19 +81,19 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 
 	/// New head subscription
 	#[pubsub(
-		subscription = "chain_finalisedHead",
+		subscription = "chain_finalizedHead",
```

Comment:
However, we don't support aliases for subscription names, so this has to use the new name despite being subscribed via alias.